event_types/event_types.py

```diff
@@ -617,6 +617,90 @@ def bin_data_in_energy(dtf, n_bins=20, log_e_reco_bins=None, return_bins=False):
         return dtf_e
 
 
+def bin_data_in_energy_and_offset(
```

Comment:
I thought this function should replace the `bin_data_in_energy` function, no? In that case, I would change the `bin_data_in_energy` function to simply call this one with `offset_bins=None`.

Reply:
I'm not sure if I understand this, because these two functions return dictionaries with different structures: for 'bin_data_in_energy' this is dtf_e > energy_range > events, while for 'bin_data_in_energy_and_offset' this is dtf_e_offset > energy_range > offset_range > events.
Do you mean that when offset_bins is None it should work as in 'bin_data_in_energy' or as in 'bin_data_in_energy_and_offset' but with one big offset bin that includes everything?

Reply:
Yes, that's what I had in mind. Either one big bin of offset or simply return the dictionary without the offset binning.

Reply:
OK, sounds good!!